string which_digit
float32[3] height_data0
float32[3] height_data1
float32[3] height_data2
float32[3] height_data3
float32[3] height_data4
float32[3] height_data5

